inductiva/__init__.py

```diff
@@ -14,6 +15,9 @@
 api_url = os.environ.get("INDUCTIVA_API_URL", "https://api.inductiva.ai")
 output_dir = os.environ.get("INDUCTIVA_OUTPUT_DIR", "inductiva_output")
 api_key = os.environ.get("INDUCTIVA_API_KEY")
+disable_filemanager_autosuffix = os.environ.get(
+    "INDUCTIVA_DISABLE_FILEMANAGER_AUTOSUFFIX", False)
+
```

Comment:
no need to have this here because it can be dynamically set (via `os.setenv`). In that case, `disable_filemanager_autosuffix` would be out of sync.